backup

```diff
@@ -7,16 +7,25 @@ get_dir() {
 }
 
 dump_database() {
-    "$(get_dir)"/postgres dump -Fc > "$BACKUP_DIRECTORY/$1"
+    if $(get_dir)/postgres dump -Fc > "$BACKUP_DIRECTORY/$1"; then
```

Comment:
C'est ou que t'as pu voir que cette syntaxe fonctionne.
On a toujours fait du if [$?...]
J'arrive pas à voir ça : https://linuxacademy.com/blog/linux/conditions-in-bash-scripting-if-statements/
J'ai peu que cela soit spécifique à une implémentation de shell

Reply:
ok

Reply:
http://tldp.org/LDP/abs/html/testconstructs.html